### ISSUE TITLE
"Unclosed Bracket" when using `url()`

### ISSUE THREAD

**Issue by ntwb** [postcss/postcss-scss#84]
The Error:

```shell
Fatal error: /pub/wporg-support/sass/modules/_wporg-header.scss:21:19: Unclosed bracket  19 | 
  20 | 		a {
> 21 | 			background: url( //s.w.org/style/images/wporg-logo.svg?3 ) center left no-repeat;
     | 			               ^
  22 | 			background-size: 290px 46px;
  23 | 			display: block;

```

This is caused by the whitespace after the opening brackets, i.e `( ` <_ The space here

❌ The following URL changes replicate the issue:
• `background: url( http://s.w.org/style/images/wporg-logo.svg?3 ) center left no-repeat;`
• `background: url( https://s.w.org/style/images/wporg-logo.svg?3 ) center left no-repeat;`
• `background: url( "//s.w.org/style/images/wporg-logo.svg?3" ) no-repeat 2px 5px;`
• `background: url( "http://s.w.org/style/images/wporg-logo.svg?3" ) no-repeat 2px 5px;`
• `background: url( "http://s.w.org/style/images/wporg-logo.svg?3" ) no-repeat 2px 5px;`

✅  The following changes can be used as workarounds:
• `background: url(//s.w.org/style/images/wporg-logo.svg?3) center left no-repeat;`
• `background: url(https://s.w.org/style/images/wporg-logo.svg?3) center left no-repeat;`
• `background: url(http://s.w.org/style/images/wporg-logo.svg?3) center left no-repeat;`

PostCSS config using grunt-postcss with Autoprefixer</summary>
```js

	var autoprefixer = require( 'autoprefixer' );
	var scsssyntax = require( 'postcss-scss' );
...
	grunt.initConfig({
		pkg: grunt.file.readJSON( 'package.json' ),
		postcss: {
			options: {
				map: false,
				processors: [
					autoprefixer({
						browsers: [ 'extends @wordpress/browserslist-config' ],
						cascade: false,
					}),
				],
				syntax: scsssyntax,
				failOnError: true,
			},
			dist: {
				expand: true,
				src: [ 'sass/**/*.scss' ],
			},
		},
...
```

**Comment by ai** [comment]
Thanks for the report. I will try to fix it on this weekend.

**Comment by ai** [comment]
Fixed 4a65819

**Comment by ai** [comment]
Released in 1.0.5

**Comment by ntwb** [comment]
Thanks 😄
